README.md

```diff
@@ -1,5 +1,227 @@
+
+## Using Kobiton plugin
+### A. Global configuration
+
+1. In the XL Release home page, go to __Settings > Shared configuration > Kobiton: Server >__    ![](./readme/button1.png)
+
+![step 2](./readme/step2.png)
+
+2. Fill all information to identify Kobiton servers
+
+    |Section|Description|
+    | -----------------| --------------- |
+    | Title            | Enter the name of your Kobiton Server<br>Example: `Kobiton` |
+    | Kobiton Username | Enter your Kobiton username |
+    | Kobiton API Key  | Enter API key of your Kobiton account |
+    | Kobiton API Url  | Enter the url of Kobiton Api: `https://api.kobiton.com`<br>For on premise or private hosting user, you can enter your host url here. |
+    | Executor Server Url | Enter the url connect to Executor Server: `https://executor.kobiton.com`<br>For on premise or private hosting user, you can enter your host url here. |
+
+![step 3](./readme/step3.png)
+
+When all fills are filled as above, you can test the connection with Kobiton Server by pressing button ![button2](./readme/button2.png).
+
+3. Only if it notices `Kobiton Server is available` as below, which means all your information is correct and XL Release can connect to Kobiton. Pleas press **Save** button to apply the config. 
+
+    ![server connected](./readme/serverconnected.png)
+
+> **Note**: 
+> 
+> - When an error is shown that cannot connect to Api or Executor server, please make sure your information is correct and you are availble to access internet. 
+>
+> - If you make everything right but cannot access to Kobiton Server, please contact us for more detail.
+
+### B. Kobiton plugin tasks
+
+* From home page choose __> Release__. Choose any release that you want to run Kobiton pluginm, if you haven't had a release yet, choose __> New release__ to create one. 
+
+![release](./readme/release.png)
+
+*  In each release we will see the Phrase widget in the release. Choose __> Add task > Manual > Kobiton__ and you will see all the Kobiton plugin there. 
+
+    ![add task](./readme/addtask.png)
+
+The fully detail of plugin will show in the below sessions.
+Kobiton Plugin now supports 3 tasks on XL Release enviroment:
+
+#### 1. List Available Devices
+This task will help to fetch all available (online) devices data that user can access. The data can pass as environment for other tasks to be consumed. 
+
+<img src="./readme/listdevices1.png" alt="listdevices1" width="600">
+
+* **Input properties**:
+
+| Field | Required | Type | Description |
+| ----- | -------- | ---- |----------- |
+|Favorite/In-house/Cloud Devices| Yes | Checkbox | The group containing devices that you want to use for automation test.<br> Note: You must choose at least 1 of these 3 options. |
```

Comment:
I suggest we should write here as `device group` because users may be confused with the `organization group`. You can contact BA for better group name